src/app.html

```diff
@@ -13,6 +13,6 @@
     %sveltekit.head%
   </head>
   <body>
-    %sveltekit.body%
+    <div style="display: contents">%sveltekit.body%</div>
```

Comment:
This was added to avoid a warning that appeared in the console when updating sveltekit